level/level.c

```diff
@@ -303,8 +306,9 @@ void destroy_level(level *lvl) {
     free((void *)lvl);
 }
 
+//TODO clearer function name?
 int rec_partition(int **room_map, int x, int y, int w, int h, int rm) {
-    if (w*h > 10*10 && rand()%100 < PARTITIONING_PERCENTAGE) {
+    if (w*h > 10*10 && rand()%100 < PARTITIONING_PROBABILITY * 100) {
```

Comment:
I'm still planning to write a little helper function that makes "dice rolls" easier to parse. That should get rid of the multiplying by 100 as well.

Reply:
Sure. Consider upgrading to a better randomness source. Every one says that rand is awful and that, in particular, the way we're using it here makes it far, far worse.

Reply:
ORLY? I would not expect it to matter outside of science.